rpm/harbour-fernschreiber.spec

```
@@ -69,7 +69,7 @@ desktop-file-install --delete-original       \
 
 %files
 %defattr(-,root,root,-)
-%{_bindir}
+%{_bindir}/*
```

```suggestion
%{_bindir}/%{name}
```

I have to admit that I've never checked that, but wouldn't that be better? My knowledge doesn't go beyond some basic stuff from the manuals, so a second opinion would be appreciated...